native-auth/config.md

```diff
@@ -9,3 +9,41 @@ dto NativeAuthClientConfig:
     gateway_url?: string;
     extra_request_headers?: dict[string, Any];
 ```
+
+## NativeAuthServerConfig
+
+```
+dto NativeAuthServerConfig:
+    // The endpoint from where the current block information will be fetched upon validation.
+    api_url: string;
```

Comment:
In `NativeAuthClientConfig`, there's also a `gateway_url`. Should we have it here, as well?

Reply:
Not sure. There isn't in the current implementation.